README.md

```diff
@@ -66,6 +66,10 @@ dependencies {
 ```
 
 ## Changelog
+*8.0.1-beta*
```

Comment:
Please revert the changes to this file and to the build.gradle. 
These files are updated automatically by the publishing script.

Reply:
Done